--- a/paths
+++ b/paths
@@ -64,13 +64,13 @@
 # file <confpaths.h> (located in include/).
 #
 
-PATH_SENDMAIL	search:sendmail:/usr/sbin:/usr/lib:/etc /usr/sbin/sendmail
-PATH_MAILDIR	d /var/mail /var/spool/mail
+PATH_SENDMAIL	search:sendmail:@MCHS_PREFIX@/bin:@MCHS_PREFIX@/lib:@MCHS_PREFIX@/etc @MCHS_PREFIX@/bin/sendmail
+PATH_MAILDIR	d @MCHS_PREFIX@/var/mail @MCHS_PREFIX@/var/spool/mail
 PATH_DEV	d /dev/
 # PATH_TTY_PFX is a prefix that's removed to get a tty device's name
 PATH_TTY_PFX	<paths.h>:_PATH_DEV d /dev/
 PATH_DEVNULL	c /dev/null
-PATH_UTMP	<utmp.h> <utmp.h>:UTMP_FILE $(localstatedir)/run/utmp search:utmp:/var/run:/var/adm:/etc
-PATH_UTMPX	<utmpx.h> <utmpx.h>:UTMPX_FILE $(localstatedir)/run/utmpx search:utmpx:/var/run:/var/adm:/etc
-PATH_WTMP	<utmp.h> <utmp.h>:WTMP_FILE $(localstatedir)/log/wtmp search:wtmp:/var/log:/var/adm:/etc
+PATH_UTMP	<utmp.h> <utmp.h>:UTMP_FILE $(localstatedir)/run/utmp search:utmp:@MCHS_PREFIX@/var/run:@MCHS_PREFIX@/var/adm:@MCHS_PREFIX@/etc
+PATH_UTMPX	<utmpx.h> <utmpx.h>:UTMPX_FILE $(localstatedir)/run/utmpx search:utmpx:@MCHS_PREFIX@/var/run:@MCHS_PREFIX@/var/adm:@MCHS_PREFIX@/etc
+PATH_WTMP	<utmp.h> <utmp.h>:WTMP_FILE $(localstatedir)/log/wtmp search:wtmp:@MCHS_PREFIX@/var/log:@MCHS_PREFIX@/var/adm:@MCHS_PREFIX@/etc
 PATH_TTY	c /dev/tty
